lib/bank_accounts/checking.rb

```diff
@@ -0,0 +1,55 @@
+require 'pry'
+
+module BankAccounts
+  class CheckingAccount < Account
+
+    attr_reader :balance, :checks_used
+
+    def initialize(id, initial_balance)
+      # inherit from Account class
+      super
+
+      # start with 0 checks used
+      @checks_used = 0
+    end
+
+    def withdraw(amount)
```

Comment:
Since there are multiple withdrawal methods it may be good to rename this one to be more specific. Is this withdrawal via ATM?